### ISSUE TITLE
PHP error: Call to undefined function bindtextdomain()

### ISSUE THREAD

**Issue by vestingz** [PodcastGenerator/PodcastGenerator#237]
**System Information**
Podcast Generator Version: v3.0.r72.gfad5f68
Webserver: Apache/2.4.41
PHP Version: PHP 7.2.26; php-gettext installed and enabled

Output of `locale -a`: 
```
C
POSIX
de_DE.utf8
en_US.utf8
```

**Podcast Generator Config**

```
<?php
$podcastgen_version = "3.1"; // Version

$first_installation = <censored>;

$installationKey = "<censored>";

$scriptlang = "en_US";

$url = "http://dummy.url.here";

$absoluteurl = "/usr/share/webapps/podcast-generator/"; // The location on the server

$theme_path = "themes/default/";

$max_upload_form_size = "104857600"; //e.g.: "30000000" (about 30MB)

$upload_dir = "media/"; // "media/" the default folder (Trailing slash required). Set chmod 755

$img_dir = "images/"; // (Trailing slash required). Set chmod 755

$feed_dir = ""; // Where to create feed.xml (empty value = root directory). Set chmod 755

$max_recent = "all"; // How many file to show in the home page

$recent_episode_in_feed = "All"; // How many file to show in the XML feed (1,2,5 etc.. or "All")

$episodeperpage = 10;

$enablestreaming = "yes"; // Enable mp3 streaming? ("yes" or "no")

$freebox = "yes"; // enable freely customizable box

$enablepgnewsinadmin = "yes";

$strictfilenamepolicy = "no"; // strictly rename files (just characters A to Z and numbers)

$categoriesenabled = "yes";

$cronAutoIndex = 1; //Auto Index New Episodes via Cron

$indexfile = "index.php";    // Path of the index file

$podcastPassword = "";       // Password to protect the podcast generator webpages, this will NOT protect the audio or XML files. Leave blank to disable.

#####################
# XML Feed stuff

$podcast_title = "Podcast Title";

$podcast_subtitle = "Subtitle";

$podcast_description = "A little description of your podcast.";

$author_name = "Podcast Generator User";

$author_email = "podcastgenerator@example.com";

$itunes_category[0] = "Arts"; // iTunes categories (mainCategory:subcategory)
$itunes_category[1] = "";
$itunes_category[2] = "";

$link = "?name="; // permalink URL of single episode (appears in the <link> and <guid> tags in the feed)

$feed_language = "en";

$copyright = "All rights reserved";   // Your copyright notice (e.g CC-BY)

$feed_encoding = "utf-8";

$explicit_podcast = "no"; //does your podcast contain explicit language? ("yes" or "no")

// END OF CONFIG

``` 

**Description of your problem:**
Error Message: 
```
[Tue Mar 24 14:32:06.453878 2020] [proxy_fcgi:error] [pid 2041056:tid 140600590669568] [client <IP>:43324] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught Error: Call to undefined function bindtextdomain() in /usr/share/webapps/podcast-generator/core/translation.php:14\nStack trace:\n#0 /usr/share/webapps/podcast-generator/core/include.php(24): include()\n#1 /usr/share/webapps/podcast-generator/index.php(11): require('/usr/share/weba...')\n#2 {main}\n  thrown in /usr/share/webapps/podcast-generator/core/translation.php on line 14\n'
<IP> - - [24/Mar/2020:14:32:06 +0100] "GET / HTTP/2.0" 500 - "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4086.0 Safari/537.36" 
```

**Comment by emilengler** [comment]
Hmmm, 3.1 will definitely include gettext as a static library inside. Nevertheless, can you change the "scriptlang" variable to "en_US.utf8"?

**Comment by vestingz** [comment]
Changed it, but it throws the same error as above :(

**Comment by emilengler** [comment]
How experienced are you with PHP and server administration?
Can you give me the output of `phpinfo();`?
Maybe a link to it or upload the .html file here

**Comment by vestingz** [comment]
[phpinfo.zip](https://github.com/PodcastGenerator/PodcastGenerator/files/4377799/phpinfo.zip)
Apparently GH does not support uploading HTML files, so here is a ZIP. I redacted a few sensitive bits.

**Comment by emilengler** [comment]
Looks like thi is a problem of your server configuration

**Comment by vestingz** [comment]
OK. Any specific ideas? It's a pretty standard setup except my php-binary, which is php72.

**Comment by emilengler** [comment]
I'm not sure if this will work but could you try `sudo phpenmod -v 7.0 gettext` and restart your webserver then

**Comment by joshmule** [comment]
I'm getting the exact same issue with version 3.1:
`Fatal error: Uncaught Error: Call to undefined function bindtextdomain() in /path/PodcastGenerator/core/translation.php:14 Stack trace: #0 /path/PodcastGenerator/core/include.php(30): include() #1 /path/PodcastGenerator/index.php(11): require('/home/cedaccom/...') #2 {main} thrown in /path/PodcastGenerator/core/translation.php on line 14`

**Comment by emilengler** [comment]
> Was gettext not included as a static library in v3.1?

Nope, I had problems founding the correct one and didn't wanted to fall for a malicious

**Comment by garyee** [comment]
I got the exact same error, my solution was that gettext php-module was not installed. Intalled it, problem solved.
